### PR TITLE
odrive version bump.

### PR DESCRIPTION
**Pull request by algogrit**

### CODE REVIEW

**Comment by gauravagarwalr** [comment]
This version bump includes support for copy app. And was a much needed bump. Please merge.

**Comment by fanquake** [comment]
Can you please squash your commits.

**Comment by gauravagarwalr** [comment]
I have squashed the commits.

File: Casks/odrive.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  url "http://cdn-mac.odrive.com/odrive.#{version}.dmg"
+  url "http://cdn-mac.odrive.com/odrivesync.#{version}.dmg"
   name 'odrive'
   homepage 'http://www.odrive.com'
   license :gratis
 
-  pkg "odrive.#{version}.pkg"
+  pkg "odrivesync.#{version}.pkg"
 
-  uninstall :pkgutil => 'com.oxygencloud.odrive.*'
```
Comment (vitorgalvao) [inline Casks/odrive.rb]: Why remove the `.`?

Comment (gauravagarwalr) [inline Casks/odrive.rb]: The namespace with the new version / old version can be either odrive or odrivesync. If a user tries to uninstall the previous version with the new cask they should be able to do so without any problems.

Comment (vitorgalvao) [inline Casks/odrive.rb]: The the `.` must stay, as it is a regex. The way you have it now it’ll just look for a bunch of `e` characters in the end.

Comment (gauravagarwalr) [inline Casks/odrive.rb]: That wasn't the intention. I thought the regex would select both com.oxygencloud.odrive.* and com.oxygencloud.odrivesync.*. I based this on the behaviour of pkgutil, which can be seen as follows:

~: $ pkgutil --pkgs=com.oxygencloud.odrive.*
com.oxygencloud.odrive.common.pkg
com.oxygencloud.odrive.installer-prod.pkg

~: $ pkgutil --pkgs=com.oxygencloud.odrive*
com.oxygencloud.odrive.common.pkg
com.oxygencloud.odrive.installer-prod.pkg

Comment (vitorgalvao) [inline Casks/odrive.rb]: Not sure what our version does, now; would have to check. Have you confirmed that uninstallation works in this cask, with your version?

Comment (gauravagarwalr) [inline Casks/odrive.rb]: I haven't sorry. Let me do this. I will reopen the pull request after verifying it.

Comment (vitorgalvao) [inline Casks/odrive.rb]: No need to close it. We can leave it open in the meantime, if you prefer.

**Pull request closed without merging**